### PR TITLE
Updated German Translation

### PR DESCRIPTION
**Pull request by jonnius**
Here we are..

### CODE REVIEW

File: locale/de/LC_MESSAGES/TaskBar.po [hunk 1]
```diff
@@ -280,160 +326,202 @@ msgstr ""
...
+"Reset the Tray Button Tab except the Icon to the Original Tray Button "
+"Settings.\n"
+"The Icon can be Reset within its own Settings."
+msgstr ""
+"Reiter Benachrichtigungssymbol zurücksetzen, außer das Symbol zu den "
+"originalen Benachrichtigungssymboleinstellungen.\n"
+"Das Symbol kann in seinen eigenen Einstellungen zurückgesetzt werden."
+
+#: prefs.js:736 prefs.js:735
+msgid "TrayButton"
```
Comment (jonnius) [inline locale/de/LC_MESSAGES/TaskBar.po]: TrayButton should be Tray Button. (line 735 in prefs.js)

File: locale/de/LC_MESSAGES/TaskBar.po [hunk 2]
```diff
@@ -144,63 +141,100 @@ msgstr ""
...
+#: prefs.js:456
+msgid "Space between Tasks (4 px)"
+msgstr "Raum zwischen Anwendungen (4 px) "
+
+#: prefs.js:464
+msgid "Reset Tasks (I) Tab"
+msgstr "Reiter Anwendungen (I) zurücksetzen"
+
+#: prefs.js:467
+msgid "Reset the Tasks Tab to the Original Tasks Settings"
```
Comment (jonnius) [inline locale/de/LC_MESSAGES/TaskBar.po]: "(I)" is missing. should be "Reset the Tasks (I) Tab to the Original Tasks Settings" (line 467 in prefs.js)

**Comment by zpydr** [comment]
Thanks for the translation and the pointers, corrected lines 467 and 735 accordingly. Vielen Dank !

**Comment by zpydr** [comment]
Hi jonnius, I have created a new TaskBar.pot file, since I am planning to release a new version. Could you translate this to German ? If yes, please take your time. Thank you.

**Pull request merged**